app/dashboard/admin.py

```diff
@@ -486,6 +487,19 @@ def explorer_link(self, instance):
         return mark_safe(f'<a href="{url}">Explorer Link</a>')
 
 
+    def response_change(self, request, obj):
+        if "_auto_update_expiry" in request.POST:
+            try:
+                bounties_to_extend = Bounty.objects.filter(event=obj)
+                for bounty in bounties_to_extend:
+                    bounty.expires_date=obj.end_date
+                    bounty.save()
```

Comment:
^ tried using bulk save here but that didn't seem to work in admin as expected hence defaulting to saving one record at a time

Reply:
There's never going to be an unreasonable amount of entries anyway, and this is an admin function so should be fine